src/app/components/course/CourseDetails/CourseDetailsBody/CourseDetailsBodyText.tsx

```diff
@@ -37,11 +37,7 @@ const CourseDetailsBodyText: React.FC<CourseDetailsBodyTextProps> = ({
     );
   });
 
-  return (
-    <div {...{ id }}>
-      <div>{paragraphs}</div>
-    </div>
-  );
+  return <div {...{ id }}>{paragraphs}</div>;
```

Comment:
Let's try to be natural and use the regular prop assignment technique. We can still use the spread operators here (i.e. "...{ id }"), but use it when it comes spontaneously, say for instance you already have an object that you want to pass here.

Another issue that I encountered, when we use spread operators, the linter fails to identify errors. For example, if we use any arbitrary attributes ("...{ id, xyz: false, someAttrThatIsNotPartOfTheInterface: false }"), you won't see the system prohibiting you from doing it. Probably we need to update the linter or configure it in a different way. Nevertheless, let's not use spread operator unless there are other incentives.

Reply:
Ok